src/asset.rs

```diff
@@ -0,0 +1,91 @@
+use std::path::PathBuf;
+use std::time::SystemTime;
+
+#[derive(Clone, Debug)]
+pub struct Data {
+    pub file: PathBuf,
+    pub last_modified: SystemTime,
+}
```

Comment:
I feel this name is too vauge. Maybe something like: `DiskData` or `FileAccess` or `LoadedData`?